plugins/woocommerce/includes/class-wc-structured-data.php

```diff
@@ -255,7 +255,7 @@ public function generate_product_data( $product = null ) {
 						'@type'      => 'AggregateOffer',
 						'lowPrice'   => wc_format_decimal( $lowest, wc_get_price_decimals() ),
 						'highPrice'  => wc_format_decimal( $highest, wc_get_price_decimals() ),
-						'offerCount' => count( $product->get_children() ),
+						'offerCount' => count( $product->get_available_variations() ),
```

Comment:
Maybe we can pass `objects` as argument to at least reduce overhead.

Reply:
@jorgeatorres great point! adjusted it per your request.